### ISSUE TITLE
Using a signature in a report.

### ISSUE THREAD

**Issue by andybut** [Joystream/founding-members#34]
Hello.
Sorry for the stupid question, but what is the purpose of https://github.com/Joystream/founding-members/blob/main/SUBMISSION-GUIDELINES.md#website-form p. 4 "sign with your membership key ..."? After all, when submitting a report, we already use json, enter the name of the member, i.e. you already receive information that is encrypted in the signature. What, in this case, does the signature provide? Now, if instead of json, password, member's name, when submitting through the form, you use a signature, then it's understandable.
And if we accept that the signature is required, then why use it for every action?
I worry when I can't understand something :).

**Comment by bwhm** [comment]
Hey,

Sorry about this, I was sure I had answered before :(

TL;DR
This is why you are required to use your key, as it's signed auto-magically when you are using the form.

There are at least two situations where you have to sign manually:
- When you are submitting via email
- When you have done something with another key (typically for validators). Unless the `stash`  key you are using for validating is the same as your membership key, the only way to know it's you is if you sign a message with `stash` saying you are `memberId`!
